pumps/prometheus_test.go

```diff
@@ -2,14 +2,16 @@ package pumps
 
 import (
 	"errors"
+	"io"
 	"testing"
 
 	"github.com/TykTechnologies/tyk-pump/analytics"
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestInitVec(t *testing.T) {
+func TestPrometheusInitVec(t *testing.T) {
```

Comment:
Updated test names in this file to have `TestPrometheus` prefix to easily be able to select tests by regex.

Reply:
makes sense @hsyed-dojo !